velox/type/Timestamp.h

```diff
@@ -104,7 +104,7 @@ struct Timestamp {
           (int64_t)(nanos_ / 1'000'000));
     } catch (const std::exception& e) {
       VELOX_USER_FAIL(
-          "Could not convert Timestamp({}, {}) to microseconds, {}",
+          "Could not convert Timestamp({}, {}) to milliseconds, {}",
```

Comment:
Thank you for the fix. It would be nice to add a test case to verify the error message.

Reply:
> Thank you for the fix. It would be nice to add a test case to verify the error message.

OK. We can reuse TimestampTest::arithmeticOverflow, but add more error message check.